module-services/service-desktop/endpoints/security/SecurityEndpointHelper.cpp

```diff
@@ -46,7 +48,7 @@ namespace sdesktop::endpoints
         return {sent::no, ResponseContext{.status = http::Code::BadRequest}};
     }
 
-    auto SecurityEndpointHelper::processStatus(Context &context) -> http::Code
+    auto SecurityEndpointHelper::processStatus(Context & /*context*/) -> http::Code
```

Comment:
why?

Reply:
Just to get rid of warning from clang-tidy about unused parameter